### PR TITLE
DDP-6402 pancan images alts

### PR DESCRIPTION
**Pull request by aglushko-broad**

### CODE REVIEW

**Comment by alextanlan** [comment]
Please fix incorrect ticket number in the description

**Comment by aglushko-broad** [comment]
> Please fix incorrect ticket number in the description

done!

File: ddp-workspace/projects/ddp-pancan/src/app/components/welcome/welcome.component.html [hunk 1]
```diff
@@ -105,7 +107,7 @@ <h2 class="title" translate>App.HomePage.StayInformedSection.Title</h2>
     </section>
 
     <section class="join-cmi">
-        <img class="join-cmi-logo" lazy-resource src="assets/images/hands.png" [alt]="'Common.Alts.JoinCMI' | translate">
+        <img class="join-cmi-logo" lazy-resource src="assets/images/hands.png" [alt]="'App.HomePage.ImageAlt' | translate">
```
Comment (alextanlan) [inline ddp-workspace/projects/ddp-pancan/src/app/components/welcome/welcome.component.html]: 'App.HomePage.ImageAlt' - not existing path, something missed there

File: ddp-workspace/projects/ddp-pancan/src/app/components/welcome/welcome.component.scss [hunk 2]
```diff
@@ -50,12 +50,8 @@ section {
         box-sizing: border-box;
 
         .image {
-            flex: 0 1 400px;
+            width: 400px;
```
Comment (alextanlan) [inline ddp-workspace/projects/ddp-pancan/src/app/components/welcome/welcome.component.scss]: Can you please fix styles for media queries (the layout is shifted)
image

image

File: ddp-workspace/projects/ddp-pancan/src/app/components/welcome/welcome.component.html (hunk 1 above)
Comment (aglushko-broad) [reply ddp-workspace/projects/ddp-pancan/src/app/components/welcome/welcome.component.html]: thanks, fixed!

File: ddp-workspace/projects/ddp-pancan/src/app/components/welcome/welcome.component.scss (hunk 2 above)
Comment (aglushko-broad) [reply ddp-workspace/projects/ddp-pancan/src/app/components/welcome/welcome.component.scss]: thanks! done

Comment (alextanlan) [reply ddp-workspace/projects/ddp-pancan/src/app/components/welcome/welcome.component.scss]: suggest changing the `width: 400px` to `width: 25rem` as to have the image fit the small (mobile) screens
(or decrease the width for mobile breakpoint)

Comment (alextanlan) [reply ddp-workspace/projects/ddp-pancan/src/app/components/welcome/welcome.component.scss]: Fixed ^ at 5c57266989ab185c5506441ff913a49258ed0f1e

**Review by alextanlan — APPROVED** [review]

**Pull request merged**